exporter/exporterparser/zipkin.go

```diff
@@ -132,6 +132,13 @@ func newZipkinExporter(finalEndpointURI, defaultServiceName, defaultLocalEndpoin
 	return zle, nil
 }
 
+func lookupAttribute(node *commonpb.Node, key string) string {
```

Comment:
personal preference: nothing against small functions (on the contrary) but I tend to add one that is used just in one place, but I am fine as it is.